pkgs/by-name/hc/hcp/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+  hcp,
+}:
+
+buildGoModule rec {
+  pname = "hcp";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    repo = "hcp";
    tag = "v${version}";
```
Avoid using pname here: #277994
`tag` is the new recommended argument. `rev` should only be used for specific commits.

---

`repo = pname;` did feel a bit weird to me but it was suggested by another reviewer. Thanks for clearing it up with the linked issue